mesos_master/assets/configuration/spec.yaml

```
@@ -0,0 +1,19 @@
+name: Mesos Master
+
+files:
+  - name: mesos_master.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/http
+          - template: init_config/default
+      - template: instances
+        options:
+          - name: url
+            description: Url used to connect to the Mesos master instance.
```

```suggestion
            description: URL used to connect to the Mesos Master instance.
```

I saw this for mesos_slave config option